models/repo.go

```diff
@@ -1402,7 +1401,17 @@ func CreateRepository(doer, u *User, opts CreateRepoOptions) (_ *Repository, err
 		}
 	}
 
-	return repo, sess.Commit()
+	err = sess.Commit()
```

Comment:
```suggestion
	if err = sess.Commit(); err != nil {
```

Reply:
Fixed. 
I'm new to Go so not used to this construct.